src/pynwb/data/nwb.file.yaml

```diff
@@ -74,27 +74,6 @@ groups:
     - neurodata_type_inc: NWBContainer
       doc: Custom analysis results
       quantity: '*'
-  - neurodata_type_inc: Epochs
-    name: epochs
-    doc: 'Experimental intervals, whether that be logically distinct sub-experiments
```

Comment:
It might be good to add this more detailed doc to the intervals/epochs specification.